packages/cubejs-schema-compiler/adapter/BaseQuery.js

```diff
@@ -1167,6 +1188,9 @@ class BaseQuery {
   }
 
   evaluateFiltersArray(filtersArray, cubeName) {
+    console.log('evaluate sql', {
```

Comment:
I believe we can remove it.

Reply:
Definitely yes. That kind of slipped through the cracks